File: IdGenerator.java
import java.util.Random;

public class IdGenerator {
    public static String createId(){

        String idNumber = "24";

        Random ran = new Random();
        for (int i =0; i<3; i++){
            int idNumber2 = ran.nextInt(10);
            idNumber+=idNumber2;
        }
        return idNumber;
    }
    public static String createBookId(){
        String idNumber = "12";
        int Booknumber;
        Random ran = new Random();
        for (int i =0; i<3; i++){
             Booknumber = ran.nextInt(10);
             idNumber+=Booknumber;
        }
        return idNumber;
    }
}

File: idgenerator.py
import random

class IdGenerator:
    def create_id(self):
        id_number = "24"
        for _ in range(3):
            id_number += str(random.randint(0, 9))
        return id_number

    def create_book_id(self):
        id_number = "12"
        for _ in range(3):
            book_number = random.randint(0, 9)
            id_number += str(book_number)
        return id_number
